api/spec/v1.yaml

```diff
@@ -2089,6 +2094,13 @@ components:
             applicable. The meaning varies based on the transaction method.
             Usually in native denomination, ParaTime units. As a string.
           example: "100000001666393459"
+        encryption_envelope:
+          description: |
+            The data relevant to the encrypted transaction. Only present for encrypted 
+            transactions in confidential EVM runtimes like Sapphire.
+            Note: The term "envelope" in this context refers to the [Oasis-style encryption envelopes](https://github.com/oasisprotocol/oasis-sdk/blob/c36a7ee194abf4ca28fdac0edbefe3843b39bf69/runtime-sdk/src/types/callformat.rs)
+            which differ slightly from [digital envelopes](https://nordvpn.com/cybersecurity/glossary/digital-envelope/).
```

Comment:
Nit: Prefer to link to [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_cryptosystem#Envelope_encryption) or an .edu domain (here's a [slightly random one](http://security.nknu.edu.tw/crypto/faq/html/2-2-4.html), probably too high-levle) over NordVPN.